### DISCUSSION TITLE
Can i do a  multiples redirection 301 with Corcel ?

### DISCUSSION THREAD

**Discussion by ChristopherMassamba** [corcel/corcel#616] [Q&A]
i migrate my old worpdress to a new Wordpress (post, page , ect...) and i would like to know if its possible  to do redirection 301 to 

redirect the old url to the new one via corcel ?

(example: old url =>"mysite.com/actualite/{slug}" to new url =>"mysite.com/actualite/{categorie}/{slug}")

**Comment by jasalt** [comment]
You could look at Redirection plugin and import CSV to it https://wordpress.org/plugins/redirection/.

Resolving WordPress permalinks from database level is not possible to my knowledge so maybe out of scope of Corcel.

You can get those from WP PHP function ’get_permalink()’ https://developer.wordpress.org/reference/functions/get_permalink/. It’s pretty simple to script it via eg. ’wp eval-file’ how you like to get data out.
